build.sh

```diff
@@ -331,10 +331,9 @@ fi
 if buildAll || hasArg cudf; then
 
     cd ${REPODIR}/python/cudf
+    python setup.py build_ext --inplace -- -DCMAKE_PREFIX_PATH=${INSTALL_PREFIX} -DCMAKE_LIBRARY_PATH=${LIBCUDF_BUILD_DIR} -DCMAKE_CUDA_ARCHITECTURES=${CUDF_CMAKE_CUDA_ARCHITECTURES} ${EXTRA_CMAKE_ARGS} -- -j${PARALLEL_LEVEL:-1}
```

Comment:
Did this undo #12945?

Reply:
Yes, that was intentional. The rebuild was happening because the scikit-build bug made Cython think that every file had been modified every time. I reverted the changes since the old approach is (very slightly) preferable until we move away from direct setup.py invocations entirely since it also avoids import errors from inside the source directory.